sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -18,8 +20,44 @@ export interface AccessPolicy {
     start: Date;
 }
 
+// @public
+export interface AccountSASOptions {
+    expiresOn?: Date;
+    ipRange?: SasIPRange;
+    permissions?: AccountSASPermissions;
```

Comment:
Feels strange this wouldn't accept `AccountSASPermissionsLike` for input, since if I have the object form I'd have to call `AccountSASPermissions.from()` just to pass it here?